ios/Video/RCTVideo.swift

```diff
@@ -1158,6 +1158,21 @@ class RCTVideo: UIView, RCTVideoPlayerViewControllerDelegate, RCTPlayerObserverH
             if _controls {
                 self.removePlayerLayer()
                 self.usePlayerViewController()
+
+                // Safely unwrap _playerViewController before applying constraints
+                if let playerViewController = _playerViewController, let playerView = playerViewController.view {
+                    playerView.translatesAutoresizingMaskIntoConstraints = false
+                    self.addSubview(playerView) // Ensure it's added to the hierarchy
```

Comment:
The idea looks good to me, but can you please move this code inside : usePlayerViewController
I am afraid is called twice addSubview(playerView), I am not sure of possible side effects

Reply:
@freeboub will test it out now, I believe you're right. Thing is, do we want the constraints to be applied only if the controls prop is true or apply it always 🤔